<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QDialog" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CreditCalc</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Сумма кредита</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineSum">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>100000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Срок кредита</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineTime">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>лет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>месяцев</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Процентная ставк</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="linePercent">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Тип ежемесячных платежей</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="radioButton_anuit">
     <property name="text">
      <string>Аннуитетные</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="radioButton_diff">
     <property name="text">
      <string>Дифференцированные</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Рассчитать</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>480</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="rowCount">
      <number>0</number>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>91</number>
     </attribute>
     <column/>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Ежемесячный платеж: </string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="pay_in_month">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Переплата по кредиту:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="overpay">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Общая выплата:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="total_pay">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Ежемесячные платежи:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineSum</tabstop>
  <tabstop>lineTime</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>linePercent</tabstop>
  <tabstop>radioButton_anuit</tabstop>
  <tabstop>radioButton_diff</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
